elixir/apps/web/lib/web/live/clients/show.ex

```diff
@@ -1,15 +1,24 @@
 defmodule Web.Clients.Show do
   use Web, :live_view
   import Web.Policies.Components
-  alias Domain.{Clients, Flows}
+  alias Domain.{Clients, Flows, Config}
 
   def mount(%{"id" => id}, _session, socket) do
     with {:ok, client} <- Clients.fetch_client_by_id(id, socket.assigns.subject, preload: :actor),
          {:ok, flows} <-
            Flows.list_flows_for(client, socket.assigns.subject,
              preload: [gateway: [:group], policy: [:resource, :actor_group]]
            ) do
-      {:ok, assign(socket, client: client, flows: flows)}
+      socket =
+        assign(
+          socket,
+          client: client,
+          flows: flows,
```

Comment:
Everything except `client` can be a temporary assign here.

Reply:
I may be overlooking something, but `client` should be able to be a temporary assign too right?

Related to temporary assigns, I've been operating under the idea that we were going to go back through and look at temporary assigns in one pass across the whole `Web` application.  Is my idea incorrect?  Would you prefer we do these types of things as one-offs?

Reply:
> I may be overlooking something, but client should be able to be a temporary assign too right?
Ah, nevermind, I see it's used in the "delete" `handle_event` function.

Reply:
Yeah, it's used in `handle_event`. We do want to go and fix all of them in one pass but also let's adopt the Boy Scout rule: always leave the camp cleaner than before. 

So when we touch some code it's better also to do minor cleanups along the way because large sweeps are very routine things to do and promote "always postponing" such cleanups.